sympy/stats/rv_interface.py

```diff
@@ -8,7 +8,7 @@
 __all__ = ['P', 'E', 'H', 'density', 'where', 'given', 'sample', 'cdf', 'characteristic_function', 'pspace',
         'sample_iter', 'variance', 'std', 'skewness', 'kurtosis', 'covariance',
         'dependent', 'independent', 'random_symbols', 'correlation',
-        'moment', 'cmoment', 'sampling_density', 'moment_generating_function', 'quantile']
+        'moment', 'cmoment', 'smoment', 'sampling_density', 'moment_generating_function', 'quantile']
```

Comment:
I recommend putting changes like this one in separate PRs. It's a different issue.